### PR TITLE
I change the name on my index.html

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
On this project I have worked on the portfolio mobile
I add  html to create my UI
I also add assets folder to store my images
And style for it to look better
My linters for the test and check the flow...

### CODE REVIEW

**Comment by KAYEMBAJONATHAN** [comment]
> On this project I have worked on the portfolio mobile I add html to create my UI I also add assets folder to store my images And style for it to look better My linters for the test and check the flow...

I have done with the linters check

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters passed ✔️
- Flexbox ✔️
- Readme file ✔️

## Required Changes ♻️
_Check the comments under the review._
### I would have made these comments inline but you have merged most of the part which makes it impossible
- [x] Your CSS is not linked to the HTML properly, I can see that your CSS name starts with a capital letter but the one you wrote is a small letter, kindly resolve it.
- [x] The logo on line 18 should have an a tag so that you will be able to add href attribute and when user clicks it, it would be able to direct them to the homepage.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KAYEMBAJONATHAN** [comment]
I HAVE FIX IT

**Comment by KAYEMBAJONATHAN** [comment]
> Hi @KAYEMBAJONATHAN,
> 
> Good job so far, indeed you have spent time on this project! There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966 ](https://camo.githubusercontent.com/e8dc1cd1bdef105014a7bd95718fbdb4a3244d39d36f9dbf8c36a8822117757a/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966) [ ](https://camo.githubusercontent.com/e8dc1cd1bdef105014a7bd95718fbdb4a3244d39d36f9dbf8c36a8822117757a/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)
> 
> ## Highlights
> * Linters passed ✔️
> * Flexbox ✔️
> * Readme file ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ### I would have made these comments inline but you have merged most of the part which makes it impossible
> * [x]  Your CSS is not linked to the HTML properly, I can see that your CSS name starts with a capital letter but the one you wrote is a small letter, kindly resolve it.
> * [x]  The logo on line 18 should have an a tag so that you will be able to add href attribute and when user clicks it, it would be able to direct them to the homepage.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

may u please review it i am done

**Comment by KAYEMBAJONATHAN** [comment]
sorry can u review

**Comment by KAYEMBAJONATHAN** [comment]
@Olamarx

**Comment by KAYEMBAJONATHAN** [comment]
can u please review 
@Olamarx

**Comment by KAYEMBAJONATHAN** [comment]
@Olamarx

**Comment by KAYEMBAJONATHAN** [comment]
> On this project I have worked on the portfolio mobile I add html to create my UI I also add assets folder to store my images And style for it to look better My linters for the test and check the flow...

I have as it required from my review @Olamarx

File: Style.css [hunk 1]
```diff
@@ -50,30 +50,29 @@ h1 {
...
+}
 
-  .union {
-   color: #6070FF;
-  }
-  
\ No newline at end of file
+.union {
+  color: #6070ff;
+}
```
Comment (Abenezer-Tilahun) [inline Style.css:78]: - [ Optional  ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust  the margins (top, right) for the hamburger button defined in the design.

The Figma design the toolbar (or header):

Capture

Your the toolbar (or header) Look likes this : 

Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @KAYEMBAJONATHAN 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions
- [x]  Good job for documenting your Read-me File Professional, But you didn't customize to according the current project you used the "Hello Microverse" Project READEME file please update it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**